README.md

```diff
@@ -19,12 +21,13 @@ within a specified VPC.
 
 ### Project origin
 
-This program was developed for the [Romana project](http://romana.io), in order
-to seamlessly deploy large [Kubernetes](https://kubernetes.io) clusters across
-multiple availability zones in an Amazon VPC. While specifically designed to
-scratch our itch for this usage scenario in the context of Romana and
-Kubernetes, the vpc-router does not depend on either project and can also be
-used stand-alone.
+This program was developed for the [Romana project](http://romana.io) to 
+overcome the limit imposed on VPC route table entries (50 by default), which limits the size of clusters.  Avoiding this limit typically required running an overlay network, which does not offer the performance and visibility of native VPC networking. 
```

Comment:
Lots of "limit" here. Maybe replace the second one with "constricts" ? As in:

    ... which constricts the size of clusters.